scss/prototype/_display.scss

```diff
@@ -0,0 +1,46 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-display
+////
+
+/// Responsive breakpoints for display classes
+/// @type Boolean
+$prototype-display-breakpoints: $global-prototype-breakpoints !default;
+
+/// Map containing all the `display` classes
+/// @type Map
+$prototype-display: (
+  none, inline, inline-block, block, table
```

Comment:
I had it flex utilities before!
From my memory, @kball told me to remove flexbox things and keep it in a separate future PR as we were planning to make flexbox first in 6.4!